docker/BuildJekyll.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:3.0.0
+
+WORKDIR /srv/jekyll
+COPY ./static/Gemfile .
+
+ARG uid
+ARG uname
+ARG gid
+ARG gname
+
+RUN groupadd -g $gid $gname
+RUN useradd -ms /bin/bash -u $uid -g $gid $uname
+
+RUN gem install jekyll bundler
+RUN bundle install
```

Comment:
I think you want to also COPY over the Gemfile.lock before doing this, too, else you'll re-run dependency resolution and again get some different results at different times.